install-scripts/linux/skm-install-linux.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
```

Comment:
**nitpick:** Perhaps we can add an empty line between the flags and the rest of the script to ease readability. WDYT?